Source/UILabelExtension.swift

```diff
@@ -60,7 +60,8 @@ public extension UILabel {
         let constraintSize = numberOfLines == 1 ?
             CGSize(width: CGFloat.greatestFiniteMagnitude, height: rectSize.height) :
             CGSize(width: rectSize.width, height: CGFloat.greatestFiniteMagnitude)
-        return binarySearch(string: string, minSize: minimumFontSize, maxSize: maxFontSize, size: rectSize, constraintSize: constraintSize)
+        let calculatedFontSize = binarySearch(string: string, minSize: minimumFontSize, maxSize: maxFontSize, size: rectSize, constraintSize: constraintSize)
+        return (calculatedFontSize * 10.0).rounded(.down) / 10.0
```

Comment:
The `rounded(.down)` isn't enough?

Reply:
That was my initial solution, but then noticed the `0.1` diff threshold which would make no sense if we then round. So respecting that I rounded to keep 1 decimal.
If you say that having effectively Integer numbers as `calculatedFontSize` then `.rounded(.down)` would be enough.

About CHANGELOG:
What version should I put there, with what release date?
I think this is something that you as the maintainer should decide.

Reply:
Got it, we can go with your solution then.
For the CHANGELOG, just add an entry below "bugfixes in the [CHANGELOG file](https://github.com/tbaranes/FittableFontLabel/blob/master/CHANGELOG.md)

Reply:
argh, sorry, completely missed the `## Next` section :D

Done.